### DISCUSSION TITLE
eleven clock not visible

### DISCUSSION THREAD

**Discussion by keitht1958** [martinet101/ElevenClock#407] [Q&A] [answered]
clock is hidden unless i hover over it tried to use settings with mouse but they disappear before i can select it
finally managed use settings by using arrow keys cannot see clock is hidden
but its still hidden windows 11 pro x64
was fine untill i put taskbar on hide but now its back to show but eleven clock is not
other windows clock shows fine
tried reinstalling but still hidden
went to settings and selected
show the clock when taskbar is hidden automatically
and it shows now
seems i solved my problem

**Comment by marticliment** [comment, marked as answer]
All right then
